### DISCUSSION TITLE
Mark merge requests with "Artifact update problem" as draft

### DISCUSSION THREAD

**Discussion by grzenkom** [renovatebot/renovate#19763] [Request Help]
I have a Python project and Renovate has just created a new MR with the following ":warning: Artifact update problem" comment:

```
Updating dependencies
Resolving dependencies...

[removed for brevity]

    So, because assistant depends on both s3fs (2022.11.0) and sagemaker (2.128.0), version solving failed.
```

It should not be merged as-is, but the pipeline passed, so GitLab is not blocking the merge and someone could miss the warning and go ahead with the merge.

Is there an option to make Renovate mark such MRs as draft, so the issue becomes more visible?

**Comment by rarkins** [comment]
This functionality is not currently possible, so please raise a feature request. Ideally include a reproduction repo on GitLab.com which you know will generate this type of failing MR.

BTW normally Renovate should add a `renovate/artifacts` failing check for the commit it made

**Comment by chopeen** [comment]
Interesting! I have seen the `renovate/artifacts` a few times, but I never know what it meant, because clicking it does not open any logs:

![image](https://user-images.githubusercontent.com/183731/211765402-a09d4c5c-3a75-4cfc-80f3-3c3cd6c4d4be.png)
Also, it is a separate pipeline and the main one succeeds, so GitLab allows to proceed with the merge.

---

I will raise a feature request with a repro repository next week.

**Reply by chopeen** [reply]
I have created a reproduction repository (~~https://github.com/chopeen/renovate-discussions-19763/pull/4~~) and now I can see how the `renovate/artifacts` should work - it should fail the pipeline.

In my case, it does not, because the main pipeline runs as an MR pipeline, while `renovate/artifacts` is a branch pipeline - therefore, they become two separate pipelines.

@rarkins Any suggestions why this may be happening?

**Reply by rarkins** [reply]
Sounds related to #13483. Renovate applies sets status checks on its commits. I don't think that is wrong, but I'm not sure why GitLab would allow you to merge a branch which has a failing pipeline.
